Dashboardui

aniswain opened the pull request:
Updated UI for upsell banner when no trained models exists

aniswain committed:
Revamped visuals of message when no models have been trained

aniswain committed:
fixed merge conflicts

karkir0003 commented on frontend/playground-frontend/src/pages/dashboard.tsx:
```diff
@@ -103,17 +112,18 @@ const BlankGrid = () => {
-          Train Model
-        </button>
-      </div>
+      <Upsell
```
@anikethswain for the PR description, please attach an image/screenshot of the upsell?

aniswain replied:
Here's a screenshot:
![image](https://github.com/DSGT-DLP/Deep-Learning-Playground/assets/41639552/4e5d4283-6e31-4098-b38d-60395adbc78c)

karkir0003 commented on frontend/playground-frontend/src/pages/dashboard.tsx:
```diff
@@ -103,17 +112,18 @@ const BlankGrid = () => {
+      <Upsell
+        imageData={{component: <Icon icon="info-circle" accessibilityLabel="" color="default" size={32}/>}}
+        message="Click on train to get started."
+        primaryAction={{
```
@anikethswain next action item could be find a way to implement something similar with material ui. 

@dwu359 thoughts on approach?
@farisdurrani for visibility

karkir0003 reviewed (commented):
conditionally approve based on @farisdurrani and @dwu359's review/comments

farisdurrani commented:
lgtm, conditionally approve awaiting Daniel's approval. You may have a commented out code at line 13 that you want to remove @anikethswain

farisdurrani commented:
Can u please share a screenshot for documentation purposes?

karkir0003 commented:
@anikethswain

dwu359 commented:
<img width="1005" alt="Screenshot 2023-05-23 at 11 08 49 PM" src="https://github.com/DSGT-DLP/Deep-Learning-Playground/assets/54150946/575cd20e-c45f-4528-af05-cdbd400ac55b">

Good enough for now, future improvements could be switching to mui (gestalt looks nice, but mui is currently more widely supported & has better docs), positioning the banner & title text/refresh button better (maybe center of screen?). Additional details that I suggest is removing the white background covering the title text and button at the top to fit the website's color scheme better, keeping the dashboard table displayed so the user expects that to be there even the 1st time they visit the website, and maybe even removing title text/refresh button entirely for a more minimalistic design (users can just use the browser's refresh button to refresh).

aniswain commented:
Thanks for attaching the screenshot!

karkir0003 commented:
@dwu359 so for this, is aniketh safe to merge this or should he address your followup comments before merging this PR in?

@anikethswain for visibility

dwu359 commented:
> @dwu359 so for this, is aniketh safe to merge this or should he address your followup comments before merging this PR in?
> 
> @anikethswain for visibility

Actually yeah pls address my comments, or if you do wish to move to a new task, we can arrange that too and the remainder of this task would be a good task for a new dev

aniswain commented:
> > @dwu359 so for this, is aniketh safe to merge this or should he address your followup comments before merging this PR in?
> > @anikethswain for visibility
> 
> Actually yeah pls address my comments, or if you do wish to move to a new task, we can arrange that too and the remainder of this task would be a good task for a new dev

I can work on addressing your comments and updating the PR

karkir0003 commented:
@dwu359 can you mark "Request Changes" on the PR so that we don't accidentally merge this in?

karkir0003 reviewed (changes requested):
@anikethswain to address @dwu359 's suggestions
